PubMed Central.js

```diff
@@ -37,7 +36,11 @@
 */
 
 function detectWeb(doc, url) {
-	if (getPMCID(url)) {
+	// Make sure the page have a PMCID and we're on a valid item page,
+	// PDF, or report page
+	if (getPMCID(url) && (url.includes(".pdf")
+	|| url.includes("/?report"))
```

Comment:
I'm not sure this is a useful check - `?report` doesn't seem to do anything, i.e., https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3139813/?report=classic and https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3139813/ are the exact same page.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC999999/?report=classic would pass this check but it 404s.

Can we get away with removing this line? Is there a particular case that it handles, like a page without `.fm-ids` but with `?report` in the URL?

Reply:
Even the `?report=reader` ebook-style view has a `.fm-ids` element.

Reply:
You're completely right -- it was for the new reader layout and I had tests failing there and assumed it didn't have the fm-ids, but I think something else was going on. Removed

Reply:
OK, great.